lib/validation.ex

```diff
@@ -7,19 +7,27 @@ defmodule SimplyValidate.Validation do
     defexception [:message]
   end
 
-  def validate(data, validators) when is_list(validators) do
-    Enum.reduce(validators, %__MODULE__{data: data}, fn validator, acc ->
-      check(acc, validator)
-    end)
+  def validate(data, validators, default_value \\ true)
+
+  def validate(data, validators, default_value) when is_list(validators) do
+    Enum.reduce(
```

Comment:
just for fun, would it be possible to make this a `Stream.transform` to allow lazy eval of the validations? might be a nice future feature.